fv3config/_asset_list.py

```diff
@@ -39,6 +39,8 @@ def ensure_is_list(asset):
 def get_orographic_forcing_asset_list(config):
     """Return asset_list for orographic forcing"""
     source_directory = get_orographic_forcing_directory(config)
+    if not len(source_directory):
+        return []
```

Comment:
Instead of writing special case code for an empty list, you can re-write this function to have the same form as `get_base_forcing_asset_list`, but using `get_orographic_forcing_directory` and using the different arguments for `asset_list_from_path`. This will enable passing an "asset list", from which all entries will be copied. If you pass an empty list, it copies no files.